### PR TITLE
Init integration test for opentelemetry plugin of shenyu-agent

### PR DESCRIPTION
**Pull request by JooKS-me**
// Describe your PR here; eg. Fixes #issueNo

Make sure that:

- [ ] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-integrated-test/shenyu-integrated-test-agent/pom.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        <groupId>org.apache.shenyu</groupId>
+        <version>2.4.2-SNAPSHOT</version>
+    </parent>
+    <packaging>pom</packaging>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>shenyu-integrated-test-agent</artifactId>
+
+    <modules>
+        <module>shenyu-integrated-test-agent-opentelemetry</module>
```
Comment (yu199195) [inline shenyu-integrated-test/shenyu-integrated-test-agent/pom.xml:32]: may be add parent pom ，like this ：

shenyu-integrated-test-agent
---------shenyu-integrated-test-agent-tracing
----------------shenyu-integrated-test-agent-opentelemetry
---------shenyu-integrated-test-agent-metrics
-----------------shenyu-integrated-test-agent-promethues
---------shenyu-integrated-test-agent-logging

File: shenyu-integrated-test/shenyu-integrated-test-agent/shenyu-integrated-test-agent-opentelemetry/src/main/resources/conf/shenyu-agent.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+supports:
+  tracing:
+    - opentelemetry
+  metrics:
+    - 
+  logging:
+    - 
+  
+plugins:
+  tracing:
```
Comment (yu199195) [inline shenyu-integrated-test/shenyu-integrated-test-agent/shenyu-integrated-test-agent-opentelemetry/src/main/resources/conf/shenyu-agent.yaml:28]: remove unused plugin

File: shenyu-integrated-test/shenyu-integrated-test-agent/shenyu-integrated-test-agent-opentelemetry/src/main/resources/conf/tracing-point.yaml [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+# limitations under the License.
+#
+
+pointCuts:
+  - targetClass: org.apache.shenyu.plugin.global.GlobalPlugin
+    points:
+      - type: instanceMethod
+        name: execute
+    handlers:
+      jaeger:
```
Comment (yu199195) [inline shenyu-integrated-test/shenyu-integrated-test-agent/shenyu-integrated-test-agent-opentelemetry/src/main/resources/conf/tracing-point.yaml:24]: remove jaeger and zipkin

File: shenyu-integrated-test/shenyu-integrated-test-agent/pom.xml (hunk 1 above)
Comment (JooKS-me) [reply shenyu-integrated-test/shenyu-integrated-test-agent/pom.xml:32]: OK, I will refactor it.

**Review by yu199195 — APPROVED** [review]

**Pull request merged**